src/main/java/com/github/manolo8/darkbot/Bot.java

```diff
@@ -32,10 +41,86 @@ public static void main(String[] args) throws IOException {
+            List<String> fileContent = Files.readAllLines(filePath, StandardCharsets.UTF_8);
+
+            if (!fileContent.isEmpty()) {
+                for (int i = 0; i < fileContent.size(); i++) {
+                    try {
+                        long filePid = 0;
+                        long fileStartTime = 0;
+                        String[] fileSplit = fileContent.get(i).split(" ");
+                        if (fileSplit.length == 2){
+                            filePid = Long.parseLong(fileSplit[0]);
+                            fileStartTime = Long.parseLong(fileSplit[1]);
+                        }
+
+                        ProcessHandle externalProcessHandle = ProcessHandle.of(filePid).orElse(null);
+                        long externalStartTime = 0;
+                        if (externalProcessHandle != null) {
+                            ProcessHandle.Info ExternalProcessInfo = externalProcessHandle.info();
+                            externalStartTime = ExternalProcessInfo.startInstant().map(Instant::toEpochMilli).orElse(0L);
+                        }
+
+                        if (ProcessHandle.of(filePid).isPresent() && externalStartTime == fileStartTime) {
+                            Files.writeString(filePath, "\n" + currentPid + " " + currentStartTime, StandardOpenOption.APPEND);
+
+                            JButton proceed = new JButton("Proceed Anyways", UIUtils.getIcon("diagnostics"));
+                            JButton cancel = new JButton("Cancel");
+                            AtomicInteger result = new AtomicInteger(-1);
+
+                            proceed.addActionListener(a -> {
+                                SwingUtilities.getWindowAncestor(proceed).setVisible(false);
+                                result.set(0);
+                            });
+                            cancel.addActionListener(a -> {
+                                SwingUtilities.getWindowAncestor(cancel).setVisible(false);
+                                result.set(1);
+                            });
+
+                            Popups.of("Multiple bots on the same folder",
+                                            "You're currently running multiple bot instances from the same folder.\n" +
+                                            "This can cause crash, unexpected problems, and some broken functionality.\n" +
+                                            "Please create a separate folder to run multiple instances.",
+                                            JOptionPane.WARNING_MESSAGE)
+                                    .options(new Object[]{proceed, cancel})
+                                    .initialValue(cancel)
+                                    .showOptionSync();
+
+                            if (result.get() == 1) {
+                                System.out.println("Reject multiple bot warning, closing bot");
+                                System.exit(100);
```

Comment:
```suggestion
                                System.exit(0);
```
This is closing the program in a normal manner, no need for an exceptional exit